CHANGELOG.md

```diff
@@ -42,6 +44,49 @@
 
 Although every attempt has been made to make this information as accurate as possible, please know there may be things that are omitted, not fully developed yet, or updates since this publication that were not included in the information below. Only the most pressing or significant items have been listed. For the entire list of tickets and or specific information about any given item, please visit the list at [Hyperleder Indy's Jira](https://jira.hyperledger.org/). Once logged in, simply navigate to Projects > Indy.
 
+## 1.9.0
+### Release date: July 04th, 2019
+
+### Component Version Information
+| Components | Version Numbers |
+| --- | --- |
+| indy-plenum | 1.9.0 |
+| indy-node | 1.9.0 |
+| sovrin | 1.1.50 |
+
+### Additional Information:
+**There are possible OOM issues during 3+ hours of target load or large catch-ups at 8 GB RAM nodes pool so 32 GB is recommended.**
+
```

Comment:
Known issue - INDY-2164: some nodes can fail to send a REJECT or REPLY to client under specific network conditions

Reply:
Done.